utils/lib/nr-graphql-helpers.ts

```diff
@@ -114,6 +114,10 @@ export const translateMutationErrors = (
     } else {
       console.error(`- ${error.message}`);
     }
+
+    if (error.message) {
+      console.error(error.message);
+    }
```

Comment:
This looks like it doubles our error logging. We probably don't need to do this.

```suggestion
```